src/test/java/com/wikia/webdriver/common/core/drivers/browsers/DefaultBrowser.java

```diff
@@ -0,0 +1,34 @@
+package com.wikia.webdriver.common.core.drivers.browsers;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.drivers.BrowserAbstract;
+import com.wikia.webdriver.common.core.drivers.BrowserType;
+
+public class DefaultBrowser extends BrowserAbstract {
+
+  private BrowserAbstract browserClass;
+
+  DefaultBrowser() {
+    try {
+      browserClass = BrowserType.lookup(Configuration.getBrowser()).getBrowserClass().newInstance();
+    } catch (InstantiationException | IllegalAccessException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)